agent/config/default.go

```diff
@@ -181,6 +181,11 @@ func DevSource() Source {
 		ports = {
 			grpc = 8502
 		}
+		
+		peering {
+			# We use peer registration for various testing
+			test_allow_peer_registrations = false
+		}
```

Comment:
We should be declaring this at `NonUserSource` only, right? Otherwise you have duplicate entries eventually merged to the final config

Reply:
Oh yes, great catch!